cmd/workspace.go

```diff
@@ -15,16 +16,18 @@ var workspaceCmd = &cobra.Command{
 
 This command can be combined with shell commands to take you there.
 
-For example, on Linux or MacOS you can run:
+For example you can run:
```

Comment:
I don't think the repetition of `workspace` sounds right, and I'd change `outputs` to `prints out`, so maybe `Print out the path to your Exercism workspace.`?